javaparser-testing/src/test/java/com/github/javaparser/printer/PrettyPrintVisitorTest.java

```diff
@@ -139,4 +139,31 @@ public void printClassWithoutJavaDocButWithComment() {
         String content = cu.toString(ignoreJavaDoc);
         assertEquals(String.format("public class A {%s    // stuff%s}%s", EOL, EOL, EOL), content);
     }
+
+    @Test
+    public void printImportsDefaultOrder() {
+        String code = "import x.y.z;import a.b.c;import static b.c.d;class c {}";
+        CompilationUnit cu = JavaParser.parse(code);
+        String content = cu.toString();
+        assertEquals(String.format("import x.y.z;%1$s" +
+                "import a.b.c;%1$s" +
```

Comment:
I saw this and was thinking: maybe we can create an assertEquals method that replaces all EOL characters with the EOL constant before calling JUnit's assertEquals? Then we can simply pass "`class X{\n}`".

Reply:
We already have a method (`com.github.javaparser.utils.Utils.normalizeEolInTextBlock()`) so it shouldn't be difficult. I can add a new pull request.

Reply:
Cool, or put it in this one.

Reply:
A new PR it is, because I'm merging this one :-)